### ISSUE TITLE
render mode is unconfigurable

### ISSUE THREAD

**Issue by fumihwh** [reactjs/react-art#54]
I have read the issue 17 and test source.
https://github.com/reactjs/react-art/issues/17
https://github.com/reactjs/react-art/blob/master/src/__tests__/ReactART-test.js#L55-L59

And I think there is no way to change the render mode.
In ReactART.js, there is requiring the art/modes/fast, that decide the mode used in ReactART.
@sebmarkbage You said that is already configurable and already do this in the unit test.
Actually L56-L59 in ReactART-test.js do nothing about ReactART Object.

I think you can just delete the L14 in ReactART.js, and let us choose the mode freely.

**Comment by webxl** [comment]
I agree with @fumihwh. The unit test has access to the dependency's API directly, whereas any application built on top of this doesn't. Mode needs to be exposed in ReactART, along with the available modes (e.g. ReactART.AvailableModes.SVG). And ideally you could change the mode with a component property.

**Comment by varghesep** [comment]
@fumihwh +

**Comment by shripadk** [comment]
+1. Would be great if the mode is configureable.

**Comment by zemlanin** [comment]
+1

**Comment by andru** [comment]
+1 Also looking to have this configurable
